capstone/src/main/java/com/google/univiz/api/SearchData.java

```diff
@@ -1,4 +1,15 @@
 package com.google.univiz.api;
 
+import com.google.auto.value.AutoValue;
+
 /** SearchData represents the college data specific to the Search feature */
-public class SearchData {}
+@AutoValue
+public abstract class SearchData {
+  public static SearchData create(String collegeName, int collegeId) {
```

Comment:
I don't think this static method needs to be public. Only api classes would ever need to create an instance of this class.
```suggestion
  static SearchData create(String collegeName, int collegeId) {
```